src/main/resources/files/readme.md.tpl

```diff
@@ -13,31 +13,24 @@ This will create an executable jar file **[# th:text="${jar_file}"/]** within th
 
     java -jar target/[# th:text="${jar_file}"/]
 
-[/]
-[# th:if="${build_tool} == 'GRADLE'"]
+[/][# th:if="${build_tool} == 'GRADLE'"]
 [# th:if="${mp_servername} == 'payara-micro'"]
     ./gradlew microBundle
-[/]
-[# th:if="${mp_servername} == 'liberty'"]
+[/][# th:if="${mp_servername} == 'liberty'"]
     ./gradlew libertyPackage
-[/]
-[# th:if="${mp_servername} == 'helidon'"]
+[/][# th:if="${mp_servername} == 'helidon'"]
     ./gradlew assemble
 [/]
 
 This will create an executable jar file **[# th:text="${jar_file}"/]** within the _build/libs_ gradle folder. This can be started by executing the following command
 
 [# th:if="${mp_servername} == 'payara-micro'"]
     ./gradlew microStart
-[/]
-[# th:if="${mp_servername} == 'liberty'"]
+[/][# th:if="${mp_servername} == 'liberty'"]
```

Comment:
can you add a paragraph on stop Liberty instance 

[/][# th:if="${mp_servername} == 'liberty'"]
When finish running Liberty in gradle, stop the backgroup processes using the following command:
./gradlew libertyStop

Reply:
@Emily-Jiang  There is already this in the file:

```
To stop Liberty running in the background, use:
```

So it would look kinda weird if I just jam in another sentence like that.  Could we move forward and fine-tune Liberty readme style in another PR?

Reply:
ah, I messed up the overall readme with quarkus readme. Please ignore my earlier comments.